testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java

```diff
@@ -34,4 +38,19 @@ public static boolean isPrestoContainer(String name)
     {
         return name.startsWith(PRESTO);
     }
+
+    public static void configureTempto(Environment.Builder builder, DockerFiles.ResourceProvider configDir, String fileDescription)
+    {
+        builder.configureContainer(TESTS, dockerContainer -> {
+            String temptoConfig = "/docker/presto-product-tests/conf/tempto/tempto-configuration-for-" + fileDescription + ".yaml";
+            dockerContainer
+                    .withCopyFileToContainer(
+                            forHostPath(configDir.getPath("tempto-configuration.yaml")),
```

Comment:
expecting `{configDir}/tempto-configuration.yaml` isn't very flexible.

we have many `tempto-configuration*.yaml` files and only some of them match this convention

```
$ find ./testing/trino-product-tests-launcher/src -name 'tempto-configuration*.yaml' | sed 's@.*/@@'
tempto-configuration.yaml
tempto-configuration.yaml
tempto-configuration-for-docker-tls.yaml
tempto-configuration-for-docker-default.yaml
tempto-configuration-for-docker-ldap.yaml
tempto-configuration-for-docker-oauth2.yaml
tempto-configuration-for-hive3.yaml
tempto-configuration-profile-config-file.yaml
tempto-configuration.yaml
tempto-configuration.yaml
tempto-configuration.yaml
```